### ISSUE TITLE
Support Interface implements Interface Syntax

### ISSUE THREAD

**Issue by iddan** [MichalLytek/type-graphql#654]
**Is your feature request related to a problem? Please describe.**
Now that [RFC: Allow interfaces to implement other interfaces](https://github.com/graphql/graphql-spec/pull/373) is merged interfaces should allow implementing other interfaces.

**Describe the solution you'd like**
Add `implements` property to `InterfaceTypeSettings`.

```typescript
@InterfaceType()
abstract class Node {}

@InterfaceType({ implements: [Node] })
abstract class Furniture extends Node {}
```

Which will generate:

```graphql
interface Node {}

interface Furniture implements Node {}
```

**Comment by MichalLytek** [comment]
It's already implemented in #602 and available in `v1.0.0-rc.2`.

**Comment by iddan** [comment]
Sorry, missed that
